packages/ui/src/proposals/components/ProposalStages.tsx

```diff
@@ -3,44 +3,73 @@ import React, { useMemo } from 'react'
+import { Tooltip } from '@/common/components/Tooltip'
 import { last, repeat } from '@/common/utils'
 import { ProposalStatus, ProposalStatusUpdates } from '@/proposals/types'
 
 export interface ProposalStagesProps extends ControlProps<number> {
-  status: ProposalStatus
+  roundStatus: ProposalStatus
   updates: ProposalStatusUpdates[]
-  constitutionality?: number | string
+  constitutionality?: number
 }
 
-export const ProposalStages = ({ status, updates, constitutionality = '-', value, onChange }: ProposalStagesProps) => {
-  const rounds = useMemo(() => {
+type RoundState = 'approved' | 'rejected' | 'deciding' | 'disabled'
+const iconMap = { approved: <CheckboxIcon />, rejected: <CrossIcon />, deciding: undefined, disabled: undefined }
+
+export const ProposalStages = ({ roundStatus, updates, constitutionality, value, onChange }: ProposalStagesProps) => {
+
+  const rounds: RoundState[] = useMemo(() => {
+
     const decidingCount = updates.filter(({ status }) => status === 'deciding').length
+
     const lastUpdate = last(updates).status
-    const onGoing = lastUpdate === status
+    const onGoing = lastUpdate === roundStatus
     const approved = lastUpdate === 'gracing'
     const rejected = !onGoing && !approved
-    const isDeciding = onGoing && status === 'deciding'
-    const isDormant = onGoing && status === 'dormant'
-    return [
-      ...repeat((round) => ({ icon: <CheckboxIcon />, onClick: () => onChange(round) }), decidingCount - 1),
-      {
-        icon: !isDeciding && (rejected ? <CrossIcon /> : <CheckboxIcon />),
-        onClick: () => onChange(decidingCount - 1),
-      },
-      ...(isDormant ? [{ icon: false, onClick: undefined }] : []),
-    ]
-  }, [updates.length, status])
+    const isDormant = onGoing && roundStatus === 'dormant'
+
+    return repeat((round) => {
+      if (round < decidingCount) {
+        return `approved`
+      } else if (round > decidingCount) {
+        return `disabled`
+      } else if (isDormant || approved) {
+        return `approved`
+      } else if (rejected) {
+        return `rejected`
+      } else {
+        return `deciding`
+      }
+    }, constitutionality ?? 1)
+  }, [updates.length, roundStatus, constitutionality])
+
 
   return (
     <TabsContainer>
-      {rounds.map(({ icon, onClick }, round) => (
-        <TabContainer key={round} active={round === value} disabled={!onClick} onClick={onClick}>
-          {icon}
-          <span>
-            Council approvals {round + 1}/{constitutionality}
-          </span>
-        </TabContainer>
-      ))}
+      {rounds.map((roundStatus, round) => {
+        const isDisabled = roundStatus === 'disabled'
+        const isActive = round === value
+        const onClick = isDisabled ? undefined : () => { return onChange(round) }
+        const icon = iconMap[roundStatus]
+        return (
+
+          <Tooltip
+            key={round}
+            tooltipText={
+              !isDisabled
```

Comment:
minor nits:
- check for `isDisabled` (not inverse)
- define strings / test earlier to make component easier to read:
>  <Tooltip key={round} tooltipText={isDisabled ? tooltip.disabled : tooltip.active}>